### PR TITLE
Add CI workflow

### PR DESCRIPTION
**Pull request by ggobugi27**
- Installs Pact with brew on ubuntu-latest
- Runs test.repl test that contains repl tests of modules. (needs to be updated with additional repl tests)

### CODE REVIEW

File: .github/workflows/applications.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Build Pact and run util REPL tests
+
+on:
+  push:
+    branches:
+    - master
```
Comment (mightybyte) [inline .github/workflows/applications.yml]: Now that I think about it, I think we can leave these branch restrictions out and have the CI job run on all branches and PRs.  We use them in some other projects when CI is slow, but that's not a problem with this project.

**Review by mightybyte — CHANGES_REQUESTED** [review]
I had one change but other than that it looks great!

**Comment by mightybyte** [comment]
Also, one other comment...have you tested this to make sure that the CI check fails and it prevents the PR from being merged when there's a test failure?  I think it would be good to test that.  You can just make a commit to this branch that has a failure and see what CI does.  Then, if it does the right thing, change it back.

File: .github/workflows/applications.yml (hunk 1 above)
Comment (ggobugi27) [reply .github/workflows/applications.yml]: Removed!

**Review by mightybyte — APPROVED** [review]

**Comment by ggobugi27** [comment]
> Also, one other comment...have you tested this to make sure that the CI check fails and it prevents the PR from being merged when there's a test failure? I think it would be good to test that. You can just make a commit to this branch that has a failure and see what CI does. Then, if it does the right thing, change it back.

**Pull request merged**